lib/components/board.ts

```diff
@@ -29,7 +30,8 @@ export const boardProps = z.object({
   outline: z.array(point).optional(),
   pcbX: distance.optional().default(0),
   pcbY: distance.optional().default(0),
-  layout: z.any().optional(),
+  layout: z.custom<LayoutBuilder>((v) => true).optional(),
+  manualEdits: z.custom<ManualEditFile>((v) => true).optional(),
```

Comment:
this is ok for now but we should move the specification for manualEdits into props and camelcasify it for the future

Reply:
https://github.com/tscircuit/props/issues/71